rspace/src/main/scala/coop/rchain/rspace/hashing/Blake2b256Hash.scala

```diff
@@ -79,27 +79,10 @@ object Blake2b256Hash {
   def fromByteString(byteString: ByteString): Blake2b256Hash =
     fromByteVector(ByteVector(byteString.toByteArray))
 
-  val codecPureBlake2b256Hash: Codec[Blake2b256Hash] = new Codec[Blake2b256Hash] {
-    val bitLength = (length * 8).toLong
-    override def decode(bits: BitVector): Attempt[DecodeResult[Blake2b256Hash]] =
-      if (bits.sizeGreaterThanOrEqual(bitLength)) {
-        Attempt.successful(
-          DecodeResult(Blake2b256Hash.create(bits.toByteVector), bits.drop(bitLength))
-        )
-      } else {
-        Attempt.failure(Err.insufficientBits(bitLength, bits.size))
-      }
-    override def encode(value: Blake2b256Hash): Attempt[BitVector] = {
-      val encoded = value.bytes.toBitVector
-      if (encoded.size != bitLength)
-        Attempt.failure(
-          Err(s"[$value] requires ${encoded.size} bits but field is fixed size of $bitLength bits")
-        )
-      else
-        Attempt.successful(encoded)
-    }
-    override def sizeBound: SizeBound = SizeBound.exact(bitLength)
-  }
+  val codecPureBlake2b256Hash: Codec[Blake2b256Hash] = scodec.codecs
```

Comment:
We had two different codec for Blake2b256Hash so `pure` was added to make a distinction, now it can be removed from the name.